extensions/wikia/SpecialDiscussionsLog/UserLogRecord.class.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/**
+ * Class UserLogRecord
+ */
+class UserLogRecord {
```

Comment:
So since this class doesn't have any behaviors, just vanilla getters/setters, it's really more of a data structure. What if we made all of these fields public and just treated it like a c-style struct? That would be cleaner IMO.

```PHP
class UserLogRecord {
	public $ip;
	public $location;
	public $language;
	public $userAgent;
	public $app;
	public $timestamp;
	public $userId;
	public $userName;
}
```